docker-compose.yml

```diff
@@ -8,6 +8,11 @@ services:
       context: .
       args:
           CONFLUENCE_VERSION: ${CONFLUENCE_VERSION}
+    environment:
+      - TZ
```

Comment:
Is there any default in here?
I think is good idea to add defaults to the .env file.
IMHO,  that gives two good things:
- Is documented
- Is asy to change even you dont really know how to work with docker environents

Reply:
I agree. The default should be the host timezone and locale. Any idea how to resolve that dynamically

Reply:
The good thing is that those variables are only passed if there is an environment variable in the host with the same name. So I am going to assume that if there is one available by default is the host one